TC_S_221 - AssertionError: 118.0 != 100

### Information about bug

======================================================================
FAIL: test_territory_wise_sales_report_TC_S_221 (erpnext.selling.report.territory_wise_sales.test_territory_wise_sales.TestTerritoryWiseSales)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.10/contextlib.py", line 79, in inner
    return func(*args, **kwds)
  File "/home/ubuntu/frappe-bench/apps/erpnext/erpnext/selling/report/territory_wise_sales/test_territory_wise_sales.py", line 53, in test_territory_wise_sales_report_TC_S_221
    self.assertEqual(row.get("quotation_amount"), 100)
AssertionError: 118.0 != 100

----------------------------------------------------------------------
Ran 1 test in 7.780s

FAILED (failures=1)

### Module

selling

### Version

Assets: v0.0.1 (pre-prod)

ERPNext: v15.59.0 (pre-prod)

ERPNext CRM: v0.0.1 (pre-prod)

ERPNext India: v0.0.1 (pre-prod)

Frappe Framework: v15.67.0 (pre-prod)

India Compliance: v15.17.1 (pre-prod)

Payments: v0.0.1 (version-15)

Projects: v0.0.1 (pre-prod)

Sales Commission: v0.0.1 (pre-prod)

### Installation method

None

### Relevant log output / Stack trace / Full Error Message.

```shell

```

--- a/erpnext/selling/report/territory_wise_sales/test_territory_wise_sales.py
+++ b/erpnext/selling/report/territory_wise_sales/test_territory_wise_sales.py
@@ -29,15 +29,21 @@ class TestTerritoryWiseSales(FrappeTestCase):
 
 		qo = make_quotation(op.name)
 		qo.items[0].warehouse = "_Test Warehouse - _TC"
+		qo.taxes_and_charges = ""
+		qo.taxes = []
 		qo.insert(ignore_permissions=True)
 		qo.submit()
 
 		so = make_sales_order(qo.name)
 		so.delivery_date = add_days(today(), 2)
+		so.taxes_and_charges = ""
+		so.taxes = []
 		so.insert(ignore_permissions=True)
 		so.submit()
 
 		si = make_sales_invoice(so.name)
+		si.taxes_and_charges = ""
+		si.taxes = []
 		si.insert(ignore_permissions=True)
 		si.submit()
 		filters = frappe._dict(

